CONTRIBUTING.md

```diff
@@ -16,15 +20,56 @@ After that run `yarn linknpm`.
 
 ### Running Unit Tests
 
-To make sure your changes do not break any unit tests, run `yarn test`. You can also run `yarn test:schematics` and `yarn test:nx` to test the schematics and nx packages individually.
+To make sure your changes do not break any unit tests, run the following:
+
+```bash
+yarn test
+```
+
+You can also run `yarn test:schematics` and `yarn test:nx` to test the schematics and nx packages individually.
 
-> If you need to test only the **collection/ngrx.spec.ts** test [for example], you can use an optional `$1` argument as follows:<br/><br/> `yarn test:schematics schematics/src/collection/ngrx/ngrx`
+If you need to test only the **collection/ngrx.spec.ts** test [for example], you can use an optional `$1` argument as follows:
+
+```bash
+yarn test:schematics schematics/src/collection/ngrx/ngrx
+```
 
 ### Running E2E Tests
 
-To make sure you changes do not break any unit tests, run `yarn e2e`. Running e2e tests can take some time, so if it often useful to run a single test. You can do it as follows: `yarn e2e lint`
+To make sure you changes do not break any unit tests, run the following:
+
+```bash
+yarn e2e
+```
+
+Running e2e tests can take some time, so if it often useful to run a single test. You can do it as follows:
+
+```bash
+yarn e2e lint
```

Comment:
This line isn't correct. It was incorrect in the original as well, but we should fix it here.